site/templates/nav.jet

```diff
@@ -2,73 +2,67 @@
 {{import "./nav/user_logged_in.jet"}}
 
 {{block nav(showSubNav=true)}}
-<div class="navigation" data-toggle="affix">
-  <nav class="navbar navbar-expand-md navbar-dark">
-      <a class="navbar-brand" href="{{ routeToPath("/") }}">{{i18n("nav_homepage")}}</a>
-      
+<div class="navigation search-hidden" data-toggle="affix" aria-hidden="true">
+  <nav>
+
+    <div class="navbar">
+      <div class="navbar-brand-container">
+        <a class="navbar-brand" href="{{ routeToPath("/") }}">{{i18n("nav_homepage")}}</a>
+      </div>
+
       {{if config("search_disabled") != "true"}}
-        <form class="form-search navbar-nav-search" action="{{ routeToPath("/search.html") }}">
-            <input class="form-control form-control-search" type="search" placeholder="{{i18n("search_control_placeholder")}}" name="q" aria-label="{{i18n("search_control_placeholder")}}">
-            <button class="sr-only" type="submit">{{i18n("search_control_submit")}}</button>
+        <form class="form-search navbar-nav-search search-desktop"  action="{{ routeToPath("/search.html") }}">
+          <button type="button" class="search-open" onclick="focusSearch()">
+            <svg class="search-icon" width="24" height="24" viewBox="0 0 24 24"><path d="M9.5,3A6.5,6.5 0 0,1 16,9.5C16,11.11 15.41,12.59 14.44,13.73L14.71,14H15.5L20.5,19L19,20.5L14,15.5V14.71L13.73,14.44C12.59,15.41 11.11,16 9.5,16A6.5,6.5 0 0,1 3,9.5A6.5,6.5 0 0,1 9.5,3M9.5,5C7,5 5,7 5,9.5C5,12 7,14 9.5,14C12,14 14,12 14,9.5C14,7 12,5 9.5,5Z" /></svg>
+          </button>
+          <input class="form-control form-control-search" type="search" placeholder="Search" onfocusout="focusOutSearch()" name="q" aria-label="{{i18n("search_control_placeholder")}}">
```

Comment:
Calling js functions directly on elements is not going to work for shift72-template sites because of the way rollup.js smushes all the js files together.  It would be better to setup an [event listener](https://developer.mozilla.org/en-US/docs/Web/API/Element/focusout_event) directly in main.js.

Reply:
Fixed. I set the event listeners in main.js